index.md

```diff
@@ -545,9 +545,9 @@ contact the <a href="mailto:featureide@uni-ulm.de">FeatureIDE development team</
 <li>Lukas Vogel (University of Magdeburg, Germany)</li>
 <li>Maik Lampe (University of Magdeburg, Germany)</li>
 <li>Malek Badeer (TU Braunschweig, Germany)</li>
-<li><a href="https://www.ict.swin.edu.au/personal/mkapuruge/">Malinda Kaushalye Kapuruge</a> (Swinburne University of Technology, Melbourne, Australia)</li>
+<li>Malinda Kaushalye Kapuruge (Swinburne University of Technology, Melbourne, Australia)</li>
 <li>Marcel Daniel (TU Braunschweig, Germany)</li>
-<li><a href="http://www.ic.ufal.br/marcio">M&aacute;rcio Ribeiro</a> (Federal University of Alagoas, Brazil)</li>
+<li>M&aacute;rcio Ribeiro (Federal University of Alagoas, Brazil)</li>
```

Comment:
This links works for me and it forwards to: https://sites.google.com/a/ic.ufal.br/marcio/